base/hmutex.h

```diff
@@ -193,7 +193,6 @@ END_EXTERN_C
 #ifdef __cplusplus
 #include <mutex>
 #include <condition_variable>
-using std::mutex;
```

Comment:
This global scope using-declaration conflicts with another `mutex` typedef in <sys/mutex.h> on Solaris. Removed as it's not used anywhere in the project.